docs/source/operations/howto/backend-deployment.rst

```diff
@@ -0,0 +1,134 @@
+        defaultDomain: https://SUBDOMAIN.HOSTDOMAIN:443
+        commonHostDomain: HOSTDOMAIN
+
+        ingress:
+          enabled: true
+          hostname: SUBDOMAIN.HOSTDOMAIN
+   
+   | Replace ``HOSTDOMAIN`` with the domain of your server without the lowest subdomain (e.g. for a server exposed at ``api.substra.org`` ``HOSTDOMAIN`` would be ``substra.org``).
+   | Replace ``SUBDOMAIN`` with the lowest subdomain (e.g. for a server exposed at ``api.substra.org`` ``SUBDOMAIN`` would be ``api``).
+
+#. Configure your connection to the orchestrator. In the ``backend-values.yaml`` file add the following content:
+
+   .. code-block:: yaml
+
+      orchestrator:
+        host: ORCHESTRATOR_HOSTNAME
+        port: ORCHESTRATOR_PORT
+        mspID: ORGANIZATION
+
+   | Replace ``ORCHESTRATOR_HOSTNAME`` with the hostname of the orchestrator.
+   | Replace ``ORCHESTRATOR_PORT`` with the port of your orchestrator (Should be ``80`` if TLS is disabled, otherwise ``443``).
+   | Replace ``ORGANIZATION`` with the name of your Organization. It should be the same value as for the ``organizationName`` key.
+
+.. _backend-channel-config:
+
+#. Configure your :term:`Substra Channels <Channel>`. 
+   In the ``backend-values.yaml`` add the following content under the ``orchestrator`` key:
+
+   .. code-block:: yaml
+
+      channels:
+        - CHANNEL:
+            restricted: RESTRICTED
+            model_export_enabled: MODEL_EXPORT
+            chaincode:
+              name: mycc
+
+   | Replace ``CHANNEL`` with the name of a channel you want to be part of, it should match one of the channels defined in your :ref:`Orchestrator Substra Channels <orchestrator-channel-config>`.
+   | Replace ``RESTRICTED`` with ``true`` if you should be the only member of this channel else ``false``.
+   | Replace ``MODEL_EXPORT`` with ``true`` if you want users from this channel to be able to download models produced by the platform, else ``false``.
+
+#. Optional: If your Orchestrator has TLS enabled:
+
+   #. Retrieve the CA certificate from your orchestrator:
+
+      The CA certificate is the ``orchestrator-ca.crt`` file generated at the :ref:`Generate your Certificate Authority certificate <orchestrator-cacert-generation>` step of the Orchestrator deployment.
+      If a public Certificate Authority was used to generate the orchestrator certificate you will need to fetch the certificate of the Certificate Authority.
+
+   #. Create a ConfigMap containing the CA certificate:
+
+      .. code-block:: bash
+
+         kubectl create configmap orchestrator-cacert --from-file=ca.crt=orchestrator-ca.crt
+
+   #. Configure your backend to enable Orchestrator TLS. In the ``backend-values.yaml`` file add the following content under the ``orchestrator`` key:
+
+      .. code-block:: yaml
+
+           tls:
+             enabled: true
+             cacert: orchestrator-cacert
+
+#. Add users to your backend. In the ``backend-values.yaml`` file add the following content:
```

Comment:
Do we have a User concept definition we can link? It might not be obvious that users are org-scoped and that it conditions their API usage.

Reply:
Good point, I'll see what I can find. Especially since now there is a user management "tool" in the frontend.